### PR TITLE
Atc fixhl7

### PR DESCRIPTION
**Pull request by ibmatc**
This will fix the HL7ToFhir conversion process. Some recent change to NAR building caused unintended classpath changes which broke HL7.  This resolves that, and addresses one conversion issue that is not yet addressed in the conversion API library.

### CODE REVIEW

**Review by ewill00 — APPROVED** [review]
LGTM

File: clinical-ingestion/helm-charts/alvearie-ingestion/values.yaml [hunk 1]
```diff
@@ -182,7 +182,7 @@ nifi:
     # The NARs will be copied to a shared volume, and the customLibPath will be set to the location
     # of that shared volume in the main NiFi server container and set in nifi.properties
     alvearie-nars-init:
-      image: alvearie/nars:0.0.2
+      image: alvearie/nars:0.0.3
```
Comment (luisgarcc) [inline clinical-ingestion/helm-charts/alvearie-ingestion/values.yaml:185]: Please remember to update the change log in Dockerhub
image

File: nifi-processors/nifi-health-patterns-bundle/nifi-health-patterns-processors/src/main/java/com/ibm/healthpatterns/processors/hl7tofhir/HL7ToFhirProcessor.java [hunk 2]
```diff
@@ -167,10 +173,33 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
...
             getLogger().error("Error converting HL7 data to FHIR", e);
             session.transfer(inputFlowFile, FAIL_RELATIONSHIP);
             return; // if the input data can't be converted to FHIR then stop
         }
     }
+
+  /**
+   * This will do some minor changes to the resulting fhir resource generated by the HL7-to-FHIR
+   * converter
+   *
```
Comment (luisgarcc) [inline nifi-processors/nifi-health-patterns-bundle/nifi-health-patterns-processors/src/main/java/com/ibm/healthpatterns/processors/hl7tofhir/HL7ToFhirProcessor.java:186]: Nit: I would mention what minor changes are needed, just to be able to know them w/o having to read the code...

**Review by luisgarcc — APPROVED** [review]
LGTM only some minor comments. 

I am also including an comment unrelated to this review but related to the HL7custom processor. I created issue that https://github.com/Alvearie/health-patterns/issues/84 to address that. Thanks!

File: nifi-processors/nifi-health-patterns-bundle/nifi-health-patterns-processors/src/main/java/com/ibm/healthpatterns/processors/hl7tofhir/HL7ToFhirProcessor.java (hunk 2 above)
Comment (ibmatc) [reply nifi-processors/nifi-health-patterns-bundle/nifi-health-patterns-processors/src/main/java/com/ibm/healthpatterns/processors/hl7tofhir/HL7ToFhirProcessor.java:190]: Updated

File: clinical-ingestion/helm-charts/alvearie-ingestion/values.yaml (hunk 1 above)
Comment (ibmatc) [reply clinical-ingestion/helm-charts/alvearie-ingestion/values.yaml:185]: Updated

**Pull request merged**